src/framework/musesampler/internal/musesamplerwrapper.cpp

```diff
@@ -45,6 +45,10 @@ MuseSamplerWrapper::MuseSamplerWrapper(MuseSamplerLibHandlerPtr samplerLib,
     m_sequencer.setOnOffStreamFlushed([this]() {
         revokePlayingNotes();
     });
+
+//    config()->samplesToPreallocateChanged().onReceive(this, [this](const samples_t samples) {
```

Comment:
is it still needed?

Reply:
Yes, there is a bug on the MuseSampler side, we're working on it. This code allows us to test it